doc/caveats.md

```diff
@@ -11,14 +11,14 @@ equivalent to creating a new commit with the edited content and discarding the
 old one. Hence, if a reply already exists on the old commit, it will not be
 removed and the reply will still refer to the old commit. However, we do not
 consider this restriction sever, especially when compared to mailing-lists,
-which inhibit the same restriction. Note that we currently still lack an tool
+which inhibit the same restriction. Note that we currently still lack a tool
```

Comment:
Not sure whether this is correct. However, I'm inclined to trust you on this.

Reply:
am sure about it, but... the best poof I could find is this:

Google finds:

* "an tool": 833.000
* "a tool": 114.000.000

Reply:
Ah. Right. Me was simply too stupid to read the diff correctly.

Reply:
ahhh :D